src/status.cc

```diff
@@ -14,6 +14,15 @@
 
 #include "status.h"
 
+#ifdef _WIN32
+#include "win32port.h"
+#else
+#ifndef __STDC_FORMAT_MACROS
+#define __STDC_FORMAT_MACROS
+#endif
+#include <inttypes.h>
```

Comment:
```
#include <cinttypes>
```

Reply:
Right, i agree that it should be `#include <cinttypes>`, but there are zero `#include <cinttypes>`,
and 5 `#include <inttypes.h>`...

Reply:
Ninja's current maintainer doesn't like changes of the nature "do a specific thing to the whole codebase", and prefers incremental "while we're here" changes.

Reply:
Well, that seems surprising to me, but i don't follow this codebase to know either way, so okay then.

Reply:
Plenty of discussion that I've had on the subject over the last few years: Most recent is here, there were other PRs where the topic is discussed, but I'm not able to find them within just a minute.
https://github.com/ninja-build/ninja/pull/2154